Make_Chocolate.rb

```diff
@@ -0,0 +1,19 @@
+=begin 
+We want make a package of goal kilos of chocolate. We have small bars (1 kilo each) and big bars (5 kilos each). Return the number of small bars to use, assuming we always use big bars before small bars. Return -1 if it can't be done.
+
+makeChocolate(4, 1, 9) → 4
+makeChocolate(4, 1, 10) → -1
+makeChocolate(4, 1, 7) → 2
+=end
+
+def makeChocolate(smallbar, bigbar, goal)
+    totalbar = smallbar + (bigbar * 5)
+    smallbar if totalbar == goal

```

Comment:
what is this line doing? if the condition is met, it is not doing anything

Reply:
So basically the math structure is, smallbar + (bigbar * 5)  
so when i calculate on this format i will be getting the goal kilos. then the if condition is used to check whether the totalbar ( calculated goal kilo) is equal to the target goal kilos (goal).